benchmarks/bioid_ner_benchmark.py

```diff
@@ -0,0 +1,520 @@
+                if ((counts['tp'] + counts['fp']) > 0) else 0
+
+            recall = counts['tp'] / (counts['tp'] + counts['fn']) \
+                if (counts['tp'] + counts['fn']) > 0 else 0
+
+            f1 = 2 * (precision * recall) / (precision + recall) \
+                if (precision + recall) > 0 else 0
+
+            results[match_type] = {
+                'precision': precision,
+                'recall': recall,
+                'f1': f1
+            }
+
+        counts_table = pd.DataFrame([
+            {
+                'Match Type': 'All Matches',
+                'True Positives': metrics['all_matches']['tp'],
+                'False Positives': metrics['all_matches']['fp'],
+                'False Negatives': metrics['all_matches']['fn']
+            },
+            {
+                'Match Type': 'Top Match',
+                'True Positives': metrics['top_match']['tp'],
+                'False Positives': metrics['top_match']['fp'],
+                'False Negatives': metrics['top_match']['fn']
+            }
+        ])
+
+        precision_recall = pd.DataFrame([
+            {
+                'Match Type': 'All Matches',
+                'Precision': results['all_matches']['precision'],
+                'Recall': results['all_matches']['recall'],
+                'F1 Score': results['all_matches']['f1']
+            },
+            {
+                'Match Type': 'Top Match',
+                'Precision': results['top_match']['precision'],
+                'Recall': results['top_match']['recall'],
+                'F1 Score': results['top_match']['f1']
+            }
+        ])
+
+        self.counts_table = counts_table
+        self.precision_recall = precision_recall
+
+        os.makedirs(RESULTS_DIR, exist_ok=True)
+        false_positives_df = pd.DataFrame(false_positives_counter.items(),
+                                          columns=['False Positive Text',
+                                                   'Count'])
+        false_positives_df = false_positives_df.sort_values(by='Count',
+                                                            ascending=False)
+        false_positives_df.to_csv(
+            os.path.join(RESULTS_DIR, 'false_positives.csv'), index=False)
+
+        print("Finished evaluating performance...")
+
+    def get_results_tables(self):
+        return self.counts_table, self.precision_recall
+
+
+def get_famplex_members():
+    from indra.databases import hgnc_client
+    fplx_entities = famplex.load_entities()
+    fplx_children = defaultdict(set)
+    for fplx_entity in fplx_entities:
+        members = famplex.individual_members('FPLX', fplx_entity)
+        for db_ns, db_id in members:
+            if db_ns == 'HGNC':
+                db_id = hgnc_client.get_current_hgnc_id(db_id)
+                if db_id:
+                    fplx_children[fplx_entity].add('%s:%s' % (db_ns, db_id))
+    return dict(fplx_children)
+
+
+fplx_members = get_famplex_members()
```

Comment:
The get_famplex_members function should be imported from the original benchmark.